### PR TITLE
Add basque translation

### PR DESCRIPTION
**Pull request by beriain**

### CODE REVIEW

File: app/src/main/res/values/arrays.xml [hunk 1]
```diff
@@ -42,6 +42,7 @@
         <item>English (day of year only)</item>
         <item>Español</item>
         <item>Italiano</item>
+        <item>Euskara</item>
```
Comment (caarmen) [inline app/src/main/res/values/arrays.xml]: Many thanks for your submission.

This particular file shouldn't be changed though. This setting for the languages is for the calendar data, which is not managed in this repository.

If you want the calendar data to be translated in basque, the translations need to be added to this repository: https://github.com/caarmen/french-revolutionary-calendar

If you only want the UI of the app to be translated into basque, the changes to this file here should be reverted.

File: app/src/main/res/values-eu/strings.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    <string name="app_name">Egutegi frantsesa</string>
+    <string name="app_name_wide">Egutegi frantsesa (horizontala)</string>
+    <string name="app_name_narrow">Egutegi frantsesa (bertikala)</string>
+    <string name="app_name_minimalist">Egutegi frantsesa (minimalista)</string>
+    <string name="app_full_name">Egutegi frantsesa</string>
+    <string name="description">Uneko data eta ordua Egutegi errepublikar frantsesean erakusten duen widgeta.</string>
+    <string name="credits">Kredituak:</string>
+    <string name="wikipedia">* Wikipediako artikulua: http://eu.wikipedia.org/wiki/Egutegi_errepublikanoa</string>
+    <string name="equinox_tool">* AstroLib C++ programa, Mark Hussek egina, urte bakoitzeko abuztuko lehen eguna zehazteko: http://mhuss.com/AstroLib/</string>
```
Comment (caarmen) [inline app/src/main/res/values-eu/strings.xml]: Does `abuztuko` refer to August or to autumn?

File: app/src/main/res/values-eu/strings.xml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    <string name="roman_numeral_algorithm">* Fraser Chapman, Erromatar zenbakizko algoritmoagatik: https://goo.gl/xw65ng</string>
+    <string name="translation_italian">* Alfonso Fusco, italiar itzulpena emateagatik.</string>
```
Comment (caarmen) [inline app/src/main/res/values-eu/strings.xml]: For info, I've created a branch `beriain-master` which adds you to the about screen credits :)

Could you please provide a translation for this string?

```xml
<string name="translation_basque">* Aitor Beriain, for providing Basque translations.</string>
````

File: app/src/main/res/values-eu/strings.xml (hunk 2 above)
Comment (beriain) [reply app/src/main/res/values-eu/strings.xml]: Good catch! My mistake, it's true ```abuztuko``` refers to August, and should be ```udazkeneko``` to refer to autumn.

File: app/src/main/res/values-eu/strings.xml (hunk 3 above)
Comment (beriain) [reply app/src/main/res/values-eu/strings.xml]: The translation would be: "* Aitor Beriain, euskarazko itzulpenengatik."

Should I do a new pull request, or what would be the most convenient way to proceed?

Thanks.

Comment (caarmen) [reply app/src/main/res/values-eu/strings.xml]: I'll incorporate the changes :)

**Comment by caarmen** [comment]
This has been merged.

**Pull request closed without merging**